app/internal/daily_quotes.py

```diff
@@ -9,22 +9,40 @@
 TOTAL_DAYS = 366
 
 
-def create_quote_object(quotes_fields: Dict[str, Optional[str]]) -> Quote:
-    """This function create a quote object from given fields dictionary.
-    It is used for adding the data from the json into the db"""
+def get_quote(data: Dict[str, Optional[str]]) -> Quote:
+    """Returns a Quote object from the dictionary data.
+
+    Args:
+        data: A dictionary quote related information.
+
+    Returns:
+        A new Quote object.
+
+    """
     return Quote(
-        text=quotes_fields['text'],
-        author=quotes_fields['author']
+        text=data['text'],
+        author=data['author'],
```

Comment:
Prefer not to use the name `data` for variable names

Reply:
Done.